scripts/clone_template.py

```diff
@@ -60,6 +60,26 @@ def parse_cmd_line():
     return args
 
 
+def destroy_vm(provider_mgmt, vm_name):
```

Comment:
<jkrocil> I dont believe it should be
<jkrocil> the script clearly depends only on get_mgmt
<jkrocil> there is no crud objects involved